contentcuration/contentcuration/static/js/edit_channel/new_channel/views.js

```diff
@@ -329,7 +329,12 @@ var ChannelListItem = BaseViews.BaseListEditableItemView.extend({
 	},
 	open_channel:function(event){
 		if(this.$('.channel-container-wrapper').hasClass('highlight')){
-			window.location.href = '/channels/' + this.model.get("id") + ((this.can_edit)? '/edit' : '/view');
+			var open_url = '/channels/' + this.model.get("id") + ((this.can_edit)? '/edit' : '/view');
+			if (event.metaKey) {
```

Comment:
To support Windows, could you change this to `event.metaKey || event.ctrlKey`?

Reply:
Oops — good catch! Didn't read the second comment on that SO answer :)